src/app/frontend/deploy/deploy_module.js

```diff
@@ -18,6 +18,8 @@ import deployLabelDirective from './deploylabel_directive';
 import deployFromFileDirective from './deployfromfile_directive';
 import fileReaderDirective from './filereader_directive';
 import uploadDirective from './upload_directive';
+import helpSectionDirective from './helpsection/helpsection_directive';
```

Comment:
Hmm... Can you move this to a separate module? I.e., so that every folder has its own module. And then depend on this module here.